authors/apps/articles/serializers.py

```diff
@@ -1,3 +1,6 @@
+from urllib.parse import quote_plus, quote
```

Comment:
@fahadmak You imported quote_plus but never used it.

Reply:
Thanx for the observation let me work on it